pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -290,6 +296,14 @@ static int guess_offsets(tracer_status_t* status, struct sock* skp) {
         }
         new_status.netns = possible_netns;
         break;
+    case GUESS_RTT:
+        possible_rtt = 0;
+        possible_rtt_var = 0;
+        bpf_probe_read(&possible_rtt, sizeof(possible_rtt), ((char*)skp) + status->offset_rtt);
+        bpf_probe_read(&possible_rtt_var, sizeof(possible_rtt_var), ((char*)skp) + status->offset_rtt_var);
+        new_status.rtt = possible_rtt;
```

Comment:
Any reason not to read this value directly into `new_status`?

Reply:
no, I was just following the same pattern present in all other cases from this big `switch` statement, but now that you brought this up I think I'll add another commit to my refactoring PR and clean up this function. there is a a good chunk of useless boilerplate code there that we can get rid of.